core/src/main/java/org/apache/iceberg/CatalogProperties.java

```diff
@@ -140,6 +140,8 @@ private CatalogProperties() {}
 
   public static final String APP_ID = "app-id";
   public static final String USER = "user";
+  public static final String ENGINE_NAME = "engine-name";
+  public static final String ENGINE_VERSION = "engine-version";
```

Comment:
Why put these in `CatalogProperties`? Wouldn't `EnvironmentContext` be more appropriate?

Reply:
both seemed appropriate, but I can move them to `EnvironmentContext`